tests/tests.py

```diff
@@ -35,85 +33,42 @@
     LABEL_LIST_ID,
 )
 
+# these tests can be run from the command line via
+# python manage.py test tests.tests --settings="tests.test_settings"
+
 
 class ViewTests(TestCase):
-    @classmethod
-    def mock_concept_and_scheme_graphs(cls):
-        """Bootstrap just a few nodes as an alternative to loading the entire package."""
+    graph_fixtures = ["scheme.json", "concept.json"]
```

Comment:
We finally outgrew this. Too bad the package takes a while to load, but such is life 🍃